package.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
The `package.json` file should use _tabs_ for indentation, not 2 spaces.

Reply:
The core `.editorconfig` file specifies spaces for `package.json` though: 

https://github.com/WordPress/wordpress-develop/blob/3ba1a2b33c71a5b8dfbc6ab728bcdbff30eb1880/.editorconfig#L16-L18

Reply:
Interesting, I didn‘t know about that limitation in npm. Figured spaces are more readable for JSON files anyway.

Since the 5.0 branch is de facto trunk right now, that‘s what I linked to.

Reply:
Right, I'm also going to backport that change for 5.0, Gutenberg switched to _tabs_ around the same time, as such I think having 5.0 and Gutenberg indentation parity would be a nice to have